tests/e2e/utils/generate.ts

```diff
@@ -0,0 +1,41 @@
+import { ng } from './process';
+import { expectFileToExist } from './fs';
+import { expectToFail } from "./utils";
+
+export interface TestGenerateParam {
+  blueprint: 'class' | 'component' | 'directive' | 'interface' | 'guard' |
+    'module' | 'pipe' | 'service';
+  name: string;
+  flags?: string[];
+  pathsToVerify?: string[];
+}
+
+export function testGenerate(
+  {blueprint, name, flags, pathsToVerify}: TestGenerateParam): Promise<any> {
+
+    flags = flags || [];
```

Comment:
Indentation is wrong. These two lines should be indented by 2, and the arguments should be by 4.